infra/event-sync/build.gradle.kts

```diff
@@ -33,7 +33,9 @@ dependencies {
     implementation(project(":infra:enrolment-records:repository"))
     implementation(project(":infra:recent-user-activity"))
 
+    implementation(libs.libsimprints)
     implementation(libs.workManager.work)
     implementation(libs.retrofit.core)
     implementation(libs.jackson.core)
+    implementation(libs.androidX.ui.preference)
```

Comment:
While I understand why it is added, are there any options to avoid adding UI package to the infra module?